vm_core.h

```diff
@@ -269,22 +273,91 @@ STATIC_ASSERT(sizeof_iseq_inline_constant_cache_entry,
                sizeof(const rb_cref_t *)) <= RVALUE_SIZE);
 
 struct iseq_inline_constant_cache {
-    struct iseq_inline_constant_cache_entry *entry;
+    union {
+        struct iseq_inline_constant_cache_entry *ext;
```

Comment:
What does "ext" stand for? "Ext"ended? I find this name confusing.

Could we just call this `extended` and rename the flag to `CONST_CACHE_HAS_EXTENDED` (if that's what it means)? Or maybe "heap" and `CONST_CACHE_IN_HEAP` (or `CONST_CACHE_EMBEDDED`).

Reply:
Yes that what it stand for. If I managed to make this work, I'll rename it as you suggested.